### PR TITLE
Update statistics documentation

### PR DESCRIPTION
**Pull request by polinaeterna**
TODO:

- [x] readme for worker
- [x] endpoint docs
- [x] update images of the viewer

### CODE REVIEW

File: docs/source/statistics.mdx [hunk 1]
```diff
@@ -0,0 +1,405 @@
+# Explore statistics over split data
+
+Datasets Server provides a `/statistics` endpoint for fetching some basic statistics precomputed for a requested dataset. This will get you a quick insight on how the data is distributed.
+
+<Tip warning={true}>
+  Currently, only <a href="./parquet">datasets with Parquet exports</a>
```
Comment (AndreaFrancis) [inline docs/source/statistics.mdx:6]: Not sure if this section is part of statistics, maybe for index?

File: docs/source/index.mdx [hunk 2]
```diff
@@ -17,19 +17,19 @@ Let Datasets Server take care of the heavy lifting so you can use a simple **RES
   <img
     style="margin-bottom: 0;"
     class="block dark:hidden"
-    src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets-server/oasst1_light.png"
+    src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets-server/openbookqa_light.png"
```
Comment (severo) [inline docs/source/index.mdx:20]: Is it me, or do the two new screenshots have a low-quality resolution?

File: docs/source/statistics.mdx [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+        }
+      }
+    }
+  ]
+}
+```
+
+## Supported data types
+
+The following column data types are currently supported:
```
Comment (severo) [inline docs/source/statistics.mdx:168]: Maybe add some details here. I think the concept of "data types" in general is not defined (it depends on the data format), and as the datasets server uses Python, in fact, we are supporting a list of Python (and `datasets`-specific) types.

So: should we say that, with a list of the exact supported Python types?

Or on the contrary only give an overview like:

> strings, float and integers numbers, and the special `ClassLabel` type defined by the `datasets` library

File: docs/source/index.mdx (hunk 2 above)
Comment (polinaeterna) [reply docs/source/index.mdx:20]: idk this is how my laptop does screenshots...

Comment (severo) [reply docs/source/index.mdx:20]: I'll try to make higher resolution screenshots and upload them to huggingface/documentation-images

Comment (severo) [reply docs/source/index.mdx:20]: done: 

https://huggingface.co/datasets/huggingface/documentation-images/blob/main/datasets-server/openbookqa_dark.png 

https://huggingface.co/datasets/huggingface/documentation-images/blob/main/datasets-server/openbookqa_light.png

Comment (severo) [reply docs/source/index.mdx:20]: I also updated https://huggingface.co/docs/hub/datasets-viewer

From

Capture d’écran 2023-10-09 à 15 26 06

To

Capture d’écran 2023-10-09 à 15 26 14

---


https://huggingface.co/datasets/huggingface/documentation-images/commit/a5db8a891e22405e0d35cc549868281a1a0d75ca

Comment (polinaeterna) [reply docs/source/index.mdx:20]: awesome thank you!

File: docs/source/statistics.mdx (hunk 3 above)
Comment (polinaeterna) [reply docs/source/statistics.mdx:166]: i like it the second way! don't want to go into details of underlying types and stuff, rewritten in [46ab3a4](https://github.com/huggingface/datasets-server/pull/1936/commits/46ab3a4d6195b6ec922084c787245ef7c14ccf52)

Comment (severo) [reply docs/source/statistics.mdx]: yes, perfectly clear now

**Review by severo — APPROVED** [review]

**Pull request merged**